components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json

```diff
@@ -158,28 +158,52 @@
         "Roles": "admin"
       },
       {
-        "Name": "apim_analytics:product_manager",
-        "Roles": "admin,Internal/creator,Internal/publisher,Internal/subscriber"
+        "Name": "apim_analytics:monitoring_dashboard:own",
+        "Roles": "admin,Internal/analytics"
       },
       {
-        "Name": "apim_analytics:api_developer",
-        "Roles": "admin,Internal/creator,Internal/publisher"
+        "Name": "apim_analytics:monitoring_dashboard:edit",
+        "Roles": "admin,Internal/analytics"
       },
       {
-        "Name": "apim_analytics:app_developer",
-        "Roles": "admin,Internal/subscriber"
+        "Name": "apim_analytics:monitoring_dashboard:view",
+        "Roles": "admin,Internal/analytics"
       },
       {
-        "Name": "apim_analytics:devops_engineer",
-        "Roles": "admin,Internal/creator,Internal/publisher,Internal/subscriber"
+        "Name": "apim_analytics:manager_dashboard:own",
+        "Roles": "admin,Internal/analytics"
       },
       {
-        "Name": "apim_analytics:analytics_viewer",
+        "Name": "apim_analytics:manager_dashboard:edit",
         "Roles": "admin,Internal/analytics"
       },
       {
-        "Name": "apim_analytics:everyone",
-        "Roles": "Internal/everyone"
+        "Name": "apim_analytics:manager_dashboard:view",
+        "Roles": "admin,Internal/analytics"
+      },
+      {
+        "Name": "apim_analytics:publisher_dashboard:own",
+        "Roles": "Internal/creator,Internal/publisher"
```

Comment:
Add `admin` role to all the scopes

Reply:
fixed with 5a3ee18224c414b9f77a23a17c9b41da6ddc1f37